src/Internal/RejectedPromise.php

```diff
@@ -8,6 +8,9 @@
 
 /**
  * @internal
+ *
+ * @template-implements PromiseInterface<T>
+ * @template-covariant T
```

Comment:
It's my understanding this should read like `RejectedPromise<never> implements PromiseInterface<never>`, no?

Should probably add some tests similar to this:

```php
assertType('React\Promise\PromiseInterface<never>', new RejectedPromise(new \RuntimeException()));
```

Not sure how much of this is supported, WDYT?

Reply:
Not sure if this is a yes or no:

```
 ------ ------------------------------------------------------------------------------------------------------ 
  Line   tests/types/reject.php                                                                                
 ------ ------------------------------------------------------------------------------------------------------ 
  6      Expected type React\Promise\PromiseInterface<never>, actual: React\Promise\PromiseInterface<*NEVER*>  
 ------ ------------------------------------------------------------------------------------------------------ 
```